website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,167 @@
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png =626x*)
+
+##### Multiple variant
+The afterOptions and footer default to visible. The buttons allow users to expedite the deselection process. The counter (selected / total) is optional. Either both or individually may be shown.
+
+![Custom content enabled before the list in both single and multiple variants](/assets/components/form/super-select/super-select-multiple-variant-footer-without-counter.png =757x*)
+
+Show selected will change the list to display only selected list items while toggling the label of that button to “Show all."
+
+![Flow showing the behavior of the show selected button and how it toggles the list to only display selected list items](/assets/components/form/super-select/super-select-flow-select-all.png =836x*)
+
+If show selected is used while no selections have been made, a string of text will let the user know that no selections have been made yet.
+
+![Flow showing the behavior of the show selected button when no selections have been made yet.](/assets/components/form/super-select/super-select-flow-no-selections-select-all.png =836x*)
+
+Additionally, “clear selected” will wipe out the selections and revert the SuperSelect to its defined default state.
+
+![Flow showing the behavior of the clear selected button and how it reverts the SuperSelect's state to the default one](/assets/components/form/super-select/super-select-flow-clear-selected.png =836x*)
+
+##### Single and multiple variant footer with custom content
+In either of the two variants, you are able to fill the footer with custom content. This is helpful when wanting to mimic the footer but still customize its contents.
+
+![Showing the footer with custom content inside.](/assets/components/form/super-select/super-select-single-multiple-footer-custom-content.png =836x*) 
+
+#### CustomContent
+If enabled, yielded content will occupy this space.
+
+![Showing the afterOptions slot with custom content](/assets/components/form/super-select/super-select-after-options-custom-content.png =836x*)
+
+## List
+
+### Positioning
+Lists can be positioned to the left, center or right of the trigger, and above or below the trigger to fit more appropriately within the UI. This defaults to the bottom left with collision detection.
+
+![Showing the different list positions (left, center and right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
+
+#### Match the trigger
+Default behavior where the list's width matches the trigger's.
+
+![Showing the list's width matching the trigger's](/assets/components/form/super-select/super-select-list-width-matching.png =504x*)
+
+#### Auto width
+If the list width is set to auto, it will size to the longest list item text string.
```

Comment:
This read a little odd, so I tried something else. It's not perfect, though. 
```suggestion
If the list width is set to auto, it will expand to be the size of the longest list item.
```

Reply:
I think this explains it well, I don't want to over think it :)